css-view-transitions-1/Overview.bs

```diff
@@ -1903,11 +1907,19 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 				Note: The above code example contains variables to be replaced.
 
+			1. Let |liveSnapshot| be  the result of [=capturing the image=] of |capturedElement|'s [=new element=].
```

Comment:
We need to account for the fact that new element can be null here.

Reply:
Done